natspec/RedeemManagerV1.md

```
@@ -271,6 +271,30 @@ Creates a redeem request
+|---|---|---|
+| _lsETHAmount | uint256 | The amount of LsETH to redeem |
+| _recipient | address | The recipient owning the redeem request |
+| _initiator | address | The initiator of the request |
+
+#### Returns
+
+| Name | Type | Description |
+|---|---|---|
+| redeemRequestId | uint32 | The id of the redeem request |
+
+### requestRedeem
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)